lib/spack/docs/getting_started.rst

```diff
@@ -600,6 +600,61 @@ flags to the ``icc`` command:
            spec: intel@15.0.24.4.9.3
 
 
+^^^^^^^^^^
+SYCL/DPC++
+^^^^^^^^^^
+
+The SYCL/DPC++ compiler is Intel and Codeplay's `staging area compiler <https://github.com/intel/llvm>`_ for upcoming Clang/LLVM SYCL support.
+They are usually distributed with `oneAPI <https://software.intel.com/content/www/us/en/develop/articles/oneapi-repo-instructions.html>`_ and compilers are activated with ``source /opt/intel/inteloneapi/setvars.sh`` on Linux.
+
+One can record these activation steps by comparing set environment variables:
+
+.. code-block:: console
+
+   env > env.txt
+   set +e
+   source /opt/intel/inteloneapi/setvars.sh
+   env > env_extras.txt
+   diff env.txt env_extras.txt
+
+All newly set environment hints will then be shown in this diff.
+A typical entry in ``compilers.yaml`` looks like this:
+
+.. code-block:: yaml
+
+   # DPC++ 2021.1-beta07 (2020.5.0.0604)
+   - compiler:
+       spec: clang@11.0.0
+       paths:
+         cc: /opt/intel/inteloneapi/compiler/latest/linux/bin/clang
+         cxx: /opt/intel/inteloneapi/compiler/latest/linux/bin/dpcpp
+         f77:
+         fc:
+       flags: {}
+       operating_system: ubuntu18.04
+       target: x86_64
+       modules: []
+       environment:
+         set:
+           OCL_ICD_FILENAMES: libintelocl_emu.so:libalteracl.so:/opt/intel/inteloneapi/compiler/latest/linux/lib/x64/libintelocl.so
```

Comment:
@becker33 do I need to be careful with the `:`s here? maybe quote all strings or so?

Reply:
I think you need to quote the strings with colons in them.